### PR TITLE
Bump Kubernetes version to  1.23.0

### PR DESCRIPTION
**Pull request by TeddyAndrieux**
- Bump Kubernetes version to 1.23.0
- Bump etcd version to 3.5.1-0
- Bump CoreDNS version to v1.8.6

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
 # CHANGELOG
 ## Release 123.0.0 (in development)
 
+### Enhancements
+
+- Bump Kubernetes version to 1.23.0
```
Comment (gdemonet) [inline CHANGELOG.md]: I know we weren't doing it before, but wondering, would it make sense to link to the upstream releases when we bump such components?

```suggestion
- Bump Kubernetes version to [1.23.0](https://github.com/kubernetes/kubernetes/releases/tag/v1.23.0)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+  (PR[#3634](https://github.com/scality/metalk8s/pull/3634))
+
+- Bump etcd version to 3.5.1-0
```
Comment (gdemonet) [inline CHANGELOG.md]:
```suggestion
- Bump etcd version to [3.5.1](https://github.com/etcd-io/etcd/releases/tag/v3.5.1)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,17 @@
...
+  (PR[#3634](https://github.com/scality/metalk8s/pull/3634))
+
+- Bump CoreDNS version to v1.8.6
```
Comment (gdemonet) [inline CHANGELOG.md]:
```suggestion
- Bump CoreDNS version to [1.8.6](https://github.com/coredns/coredns/releases/tag/v1.8.6)
```

File: salt/metalk8s/kubernetes/kubelet/standalone.sls [hunk 4]
```diff
@@ -70,7 +70,13 @@ Create kubelet config file:
         httpCheckFrequency: 0s
         imageMinimumGCAge: 0s
         kind: KubeletConfiguration
-        logging: {}
+        logging:
+          flushFrequency: 0
+          options:
+            json:
+              infoBufferdockeSize: "0"
```
Comment (gdemonet) [inline salt/metalk8s/kubernetes/kubelet/standalone.sls]: Is this variable name correct? Looks like a typo, but I guess you just copy-pasted the kubeadm manifests?

**Comment by gdemonet** [comment]
Would be a nice project to start automating such PRs using GitHub Actions :wink:

File: salt/metalk8s/kubernetes/kubelet/standalone.sls (hunk 4 above)
Comment (TeddyAndrieux) [reply salt/metalk8s/kubernetes/kubelet/standalone.sls]: Right it's "infoBufferSize" (likely tried to hit docker in a wrong tab :smile: )

**Review by gdemonet — APPROVED** [review]

**Pull request merged**